Updates to SIW docs (#2745)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md

<ApiLifecycle access="ie" /><br>
<ApiLifecycle access="Limited GA" /><br>

> This guide covers the Javascript methods that are deprecated from the Sign-In Widget and describes how to use the `showSignIn()` method instead.

Reviewer: ```suggestion
This guide covers the Javascript method that is deprecated from the Sign-In Widget and describes how to use the `showSignIn()` method instead.
```
We left the title plural in case other deprecated methods come up. But the guide, as it is right now, only covers one deprecated method.
